tests/unit/lms/services/vitalsource/client_test.py

```diff
@@ -68,9 +71,67 @@ def test_it_signs_lti_launch_form_params(self, svc, lti_user):
             "oauth_version": "1.0",
         }
 
+    def test_api_get(self, svc, http_service):
+        svc._api_get("endpoint/path")  # pylint: disable=protected-access
```

Comment:
I guess you could move the `_api_get()` method into a separate helper class. The helper class could just be inline within the `vitalsource.py` file and could have its own unit tests. Then you could avoid the `protected-access` here.

Or alternatively you could just make `_api_get()` a public method even though nothing outside of `VitalSourceService` actually calls it. In theory it might be useful for a view to call `VitalSourceService.get()` directly one day, so that you don't _have_ to write a new `VitalSourceService` method for every single API call if you don't really need to.

I think the `disable=protected-access` is also a perfectly reasonable solution here though.

Reply:
I went for the public method. Using `.get` on the view is something we might need :+1: